src/main/java/org/photonvision/lib/PhotonCamera.java

```diff
@@ -43,6 +47,22 @@ public SimplePipelineResult getLastResult() {
     return ret;
   }
 
+  /**
+   * Sets whether the camera is in driver mode.
+   * @param driverMode Whether the camera should be in driver mode.
+   */
+  public void setDriverMode(boolean driverMode) {
+    driverModeEntry.setBoolean(driverMode);
+  }
+
+  /**
+   * Sets the pipeline index number.
+   * @param index The pipeline index number.
+   */
+  public void setPipelineIndex(int index) {
+    pipelineIndexEntry.setNumber(index);
+  }
```

Comment:
Do we want to add getters for these too? Not sure if it makes sense to but it might be useful.